string table_id
---
bool success
float64 total
menu_item[] items
string message